docs/concepts/index.asciidoc

```diff
@@ -115,6 +115,14 @@ Following are some example KQL queries.  For more detailed examples, refer to <<
 | `machine.os:win*`
 |===
 
+[float]
+[[autocomplete-suggestions]]
+==== Suggestions for autocomplete
+
+Beginning in 7.14, Kibana uses the https://www.elastic.co/guide/en/elasticsearch/reference/current/search-terms-enum.html[*terms enum API*] for autocomplete. Kibana returns results faster, but suggestions are approximate, sorted alphabetically, and can be outside the selected time range, even if `autocomplete:useTimeFilter` is enabled (as the terms enum API applies time filtering on an index-level, rather than document-level).
```

Comment:
```suggestion
Beginning in 7.14, {kib} uses the {ref}/search-terms-enum.html[terms enum API] for autocomplete. {kib} returns results faster, but suggestions are approximate, sorted alphabetically, and can be outside the selected time range, even if `autocomplete:useTimeFilter` is enabled (as the terms enum API applies time filtering on an index-level, rather than document-level).
```